client/src/Movies/MovieList.js

```diff
@@ -23,32 +25,11 @@ export default class MovieList extends Component {
   render() {
     return (
       <div className="movie-list">
-        {this.state.movies.map(movie => (
-          <MovieDetails key={movie.id} movie={movie} />
+      {this.state.movies.map(movie => (
+          <MovieCard key={movie.id} movie={movie} />
```

Comment:
Nice work here! Remember you can also use this `<MovieCard/>` component in Movies.js to make your code more DRY :)